rosidl_generator_py/package.xml

```diff
@@ -37,6 +37,8 @@
   <exec_depend>rosidl_runtime_c</exec_depend>
   <exec_depend>rpyutils</exec_depend>
 
+  <test_depend>ament_cmake_flake8</test_depend>
+  <test_depend>ament_cmake_pep257</test_depend>
```

Comment:
Ah, oops I wasn't requesting any changes. I think you were correct at first that these need to be `exec_depend` to make sure downstream message packages depend on them. 

I was only commenting to say the current package.xml depend declarations don't cover this case. We would need to create one, probably called `test_export_depend`, to support it better.

Reply:
Got it, reverted.

https://github.com/ros2/rosidl_python/pull/199#discussion_r1264225848
> Adding them as `exec_depend` means that they'll always be installed when a message package is installed, but these dependencies are probably light enough that this is fine.

It's a bit surprising no one (especially from those who are making products) has requested `test_export_depend`-like functionality. I can think of huge `test_depend`-ed packages in an upstream package.